garak/generators/base.py

```diff
@@ -74,11 +76,17 @@ def generate(self, prompt: str) -> List[str]:
 
         self._pre_generate_hook()
 
+        if generations_this_call == -1:
```

Comment:
Should this be `<= -1` instead? If somehow `-2` or whatever gets specified, we'd end up in a weird place.